doc/release-notes.md

```diff
@@ -86,6 +86,12 @@ New settings
 Updated settings
 ----------------
 
+- The meaning of the command line option `-persistmempool` (without a value
+  provided) incorrectly disabled mempool persistance. `-persistmempool` is now
+  treated like other boolen options to mean `-persistmempool=1`.
+  `-persistmempool=0`, `-persistmempool=1` and `-nopersistmempool` are
```

Comment:
```suggestion
  Passing `-persistmempool=0`, `-persistmempool=1`, and `-nopersistmempool` is
```

Reply:
- [ ] Why. []()

Reply:
![IMG_20210922_213948.jpg](https://user-images.githubusercontent.com/86518179/134533134-417e93a1-77fb-446a-8e36-52e76a8b5b53.jpg)

Reply:
- [ ] Tothemoon💯🐎📨🆔⭕🦆

Reply:
But it's the opposite always isn't it?

Reply:
Done